apollo-runtime-kotlin/src/commonMain/kotlin/com/apollographql/apollo/ApolloCall.kt

```diff
@@ -8,7 +8,7 @@ import kotlinx.coroutines.flow.Flow
 
 interface ApolloCall<D: Operation.Data> {
   @ApolloExperimental
-  fun execute(executionContext: ExecutionContext = ExecutionContext.Empty): Flow<Response<D>>
+  fun execute(): Flow<Response<D>>
```

Comment:
why this was removed?

Reply:
It wasn't used anywhere. I can add it back but if we do, I think we should clarify the precedence rules of ExecutionContext. Given that the API is loosely types, there's some footgun potential here. From my understanding, we have:

* Request ExecutionContext
* Response ExecutionContext
* Client ExecutionContext
* potentially Call ExecutionContext? (this one)

Should we document how each one is going to be used?

Reply:
I think as we are in active dev, everything can be changed as we don't have final stable API. So I guess we will document this later